ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands3.java

```diff
@@ -91,8 +91,8 @@ public void testRenameTable() throws Exception {
             "s/delta_0000001_0000001_0000/bucket_00000_0"},
         {"{\"writeid\":2,\"bucketid\":536870913,\"rowid\":0}\t4\t6",
             "s/delta_0000002_0000002_0001/bucket_00000_0"}};
-    checkResult(expected, testQuery, false, "check data", LOG);
-
+    List<String> rs = runStatementOnDriver(testQuery);
```

Comment:
should the `checkResult` run the query and do the validation?

Reply:
`checkresults` also checks whether mappers are vectorized. It is not required here.
We can extract it to a new method but in that case I would rename the current `checkresults` to something like `checkResultsAndVectorization` first but it is widely used.
WDYT?

Reply:
maybe we can keep overloaded version without vectorization check, however, I am ok with rename as well

Reply:
Added `checkResultsAndVectorization` and `checkresults`.  The first one calls `checkresults`.

Reply:
👍